v20.1/inverted-indexes.md

```diff
@@ -4,14 +4,18 @@ summary: Inverted indexes improve your database's performance and usefulness by
 toc: true
 ---
 
-Inverted indexes improve your database's performance by helping SQL locate the schemaless data in a [`JSONB`](jsonb.html) column.
+An inverted index stores a mapping from a value to its location in a schemaless data structure.  Inverted indexes are commonly used in [document retrieval systems](https://en.wikipedia.org/wiki/Document_retrieval).
 
-{{site.data.alerts.callout_success}}For a hands-on demonstration of using an inverted index to improve query performance on a <code>JSONB</code> column, see the <a href="demo-json-support.html">JSON tutorial</a>.{{site.data.alerts.end}}
+CockroachDB stores the contents of the following data types in inverted indexes:
 
+- [JSONB](jsonb.html)
+- <span class="version-tag">New in v20.1:</span> [Arrays](array.html)
+ 
+{{site.data.alerts.callout_success}}For a hands-on demonstration of using an inverted index to improve query performance on a <code>JSONB</code> column, see the <a href="demo-json-support.html">JSON tutorial</a>.{{site.data.alerts.end}}
 
 ## How do inverted indexes work?
 
-Standard [indexes](indexes.html) work well for searches based on prefixes of sorted data. However, schemaless data like [`JSONB`](jsonb.html) cannot be queried without a full table scan, since it does not adhere to ordinary value prefix comparison operators. `JSONB` needs to be indexed in a more detailed way than what a standard index provides. This is where inverted indexes prove useful.
+Standard [indexes](indexes.html) work well for searches based on prefixes of sorted data. However, schemaless data like [`JSONB`](jsonb.html) or [arrays](array.html) cannot be queried without a full table scan, since they do not adhere to ordinary value prefix comparison operators. `JSONB` in particular needs to be indexed in a more detailed way than what a standard index provides. This is where inverted indexes prove useful.
 
 Inverted indexes filter on components of tokenizable data. The `JSONB` data type is built on two structures that can be tokenized:
```

Comment:
Tokenizable is better than schemaless...

Reply:
Updated all mentions of "schemaless" to use "tokenizable" or in some cases just say "data like JSON ..."